data/brands/amenity/car_rental.json

```
@@ -6,6 +6,17 @@
     }
   },
   "items": [
+    {
+      "displayName": "U-Save",
+      "locationSet": {"include": ["001"]},
```

```suggestion
      "locationSet": {"include": ["cr", "do", "jm", "mt", "mx", "nl", "ro", "us"]},
```
per https://www.usave.com/LocationIntList.aspx